src/i18n/MControlCenter_es.ts

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="es_ES" sourcelanguage="es_ES">
+<context>
+    <name>MainWindow</name>
+    <message>
+        <source>Info</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>EC Version:</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>EC Build:</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>Carga de la batería:</source>
+        <translation type="unfinished"></translation>
```

Comment:
```suggestion
        <source>Battery charge:</source>
        <translation>Carga de la batería:</translation>
```

Reply:
Hello
The translated strings should look like this.

Reply:
Hi Dmitry.

I have updated my file.